ci/infra/testrunner/README.md

```diff
@@ -1,63 +1,213 @@
+* plugin_dir: directory used for retrieving terraform plugins. If not set, plugins are installed using terraform [discovery mechanism](https://www.terraform.io/docs/extend/how-terraform-works.html#discovery)
+* retries: maximum number of attempts to recover from failures during terraform provisioning 
+* stack name: the unique name of the platform stack on the shared infrastructure, used as prefix by many resources such as networks, nodes, among others. If not specified, the `username` is used.
+* tfdir: path to the terraform files. Testrunner must have writing permissions to this directory. Defaults to `skuba.srcpath/ci/infra`.
+* tfvars: name of the terraform variables file to be used. Defaults to "terraform.tfvars.json.ci.example"
+
+Example
 ```
-workspace: "" # Working directory for testrunner
-...
 terraform:
-  stack_name: "" # name of the stack
+  stack_name="my-test-stack"
+```
+
+#### Openstack
+
+* openrc: path to the environment setup script
+
+Example:
 ```
+openstack:
+  openrc: "/home/myuser/my-openrc.sh"
+```
+
+#### VMware
 
-You can also set things like where the skuba binary and src are
+* env_file: path to environment variables file
+
+Example:
 ```
-SKUBA_BINPATH=/path/to/bin/skuba
-SKUBA_SRCPATH=/path/to/src/skuba
+vmware: 
+   env_file: "/path/to/env/file"
 ```
 
-or
+### Skuba
+
+The Skuba section defines the location and execution options for the `skuba` command. As `testrunner` can be used either from a local development or testing environment or a CI pipeline, the configuration allows to define the location of both the source and the binary. Please notice that the source location is used as default location for other configuration elements, such as terraform files, even if the `skuba` binary is specified. 
+
+* binpath: path to skuba binary
+* srcpath: path to skuba source. Used to locate other resources, like terraform configuration, and ssh keys.
+* verbosity: verbosity level for skuba command execution
 
 ```
 skuba:
-  binpath: "" # path to skuba binary
-  srcpath: "" # path to skuba source
+  binpath: "/usr/bin/"
+  srcpath: "/go/src/github.com/SUSE/skuba"
+  verbosity: 5
+```
+
+### Log
+
+Testrunner sends output to both a console and file logger handlers, configured using the following `log` variables:
+
+* file: name of the file used to send a copy of the log with verbosity `DEBUG`. This file is located under the `workspace` directory.
+* level: debug verbosity level to console. Can be any of `DEBUG`, `INFO`, `WARNING`, `ERROR`. Defaults to `INFO`.
+* overwrite: boolean that indicates if the content of the log file must be overwritten (`True`) or log entries must be appended at the end of the file if it exists. Defaults to `False` (do not overwrite) 
+* quiet: boolean that indicates if `testrunner` will send any output to console (`False`) or not will execute silently (`True`). Quiet mode is useful when `testrunner` is used as a library. Defaults to `False`.
```

Comment:
Should it be indicated anywhere that the "True/False" booleans need to be in that specific case?  And, now that I type that, _are_ they actually case-sensitive in the YAML import? :D